sites/main-site/src/content/blog/2024/2024-07-02-new-website-structure.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: new website structure
```

Comment:
```suggestion
title: Faster nf-core website builds with a new back-end setup
```

Reply:
Don't love this, but wonder if the original could be tweaked to give a bit more nuance. Given that most people won't see any difference in structure (eg. navigation).

Reply:
Will compromise on "Faster nf-core website builds with a new code setup" 🙂